src/gui/systray.h

```diff
@@ -41,7 +41,7 @@ class AccessManagerFactory : public QQmlNetworkAccessManagerFactory
 
 #ifdef Q_OS_MACOS
 void setUserNotificationCenterDelegate();
-void checkNotificationAuth();
+void checkNotificationAuth(bool provisionalAuth = true);
```

Comment:
please do not use a bool for that as it will be hard to understand what
`checkNotificationAuth(true);` means when compared to `checkNotificationAuth(false);`

Reply:
Good point, changed to enum